### PR TITLE
Clarify SMTP configs (client / server)

### PR DESCRIPTION
**Pull request by paescuj**
Especially to clarify that `EMAIL_SMTP_NAME` is for the client

### CODE REVIEW

File: docs/self-hosted/config-options.md [hunk 1]
```diff
@@ -952,16 +952,16 @@ Based on the `EMAIL_TRANSPORT` used, you must also provide the following configu
...
-| `EMAIL_SMTP_HOST`       | SMTP Host        | --            |
-| `EMAIL_SMTP_PORT`       | SMTP Port        | --            |
-| `EMAIL_SMTP_USER`       | SMTP User        | --            |
-| `EMAIL_SMTP_PASSWORD`   | SMTP Password    | --            |
-| `EMAIL_SMTP_POOL`       | Use SMTP pooling | --            |
-| `EMAIL_SMTP_SECURE`     | Enable TLS       | --            |
-| `EMAIL_SMTP_IGNORE_TLS` | Ignore TLS       | --            |
+| Variable                | Description          | Default Value |
+| ----------------------- | -------------------- | ------------- |
+| `EMAIL_SMTP_NAME`       | SMTP client hostname | --            |
```
Comment (br41nslug) [inline docs/self-hosted/config-options.md:957]: Should this option be on top? It is an optional value that should not be filled in most cases (your server hostname should be set correctly in a proper setup)

**Review by azrikahar — APPROVED** [review]

File: docs/self-hosted/config-options.md
Comment (br41nslug) [inline docs/self-hosted/config-options.md:1]: Should we mark which options are optional for clarity?

**Review by licitdev — APPROVED** [review]
👍 `EMAIL_SMTP_NAME` is an optional client hostname.
Ref: https://nodemailer.com/smtp/#connection-options

File: docs/self-hosted/config-options.md (hunk 1 above)
Comment (paescuj) [reply docs/self-hosted/config-options.md]: Move to bottom 👍

File: docs/self-hosted/config-options.md
Comment (paescuj) [reply docs/self-hosted/config-options.md:1]: Yes, but let's do that separately (and in the best case for all options) 👍

**Pull request merged**